lab-bulma-components/src/coolButton/CoolButton.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import "../App.css";
+import "bulma/css/bulma.css";
+
+const CoolButton = (props) => {
+  return (
+    <>
+      <button class={props.className}>{props.children}</button>
```

Comment:
```suggestion
      <button className={props.className}>{props.children}</button>
```
Para especificar una clase CSS, usa el atributo className.